### PR TITLE
Add SLE12 SP2 job for s390x worker

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-crowbar_register-template.yaml [hunk 1]
```diff
@@ -18,6 +18,7 @@
           predefined-parameters: |
             TESTHEAD=1
             cloudsource=develcloud{version}
+            want_sles12sp2=1
```
Comment (jdsn) [inline scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-crowbar_register-template.yaml]: This also sets `want_sles12sp2=1` also for non s390x jobs.
Is this wanted?

**Comment by jdsn** [comment]
:+1:  if the change I commented is meant that way

File: scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-crowbar_register-template.yaml (hunk 1 above)
Comment (dirkmueller) [inline scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-crowbar_register-template.yaml]: well, it probably breaks the cloud6 register job..

Comment (jdsn) [inline scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-crowbar_register-template.yaml]: @dirkmueller well, you could still add something like this to the template:
```
 project:
   name: cloud-mkcloud7-s390x
   version: 7
   previous_version: 6
   arch: s390x
   want-sles12sp2=1
   label: openstack-mkcloud-SLE12-{arch}
   jobs:
       - 'cloud-mkcloud{version}-job-crowbar_register-{arch}'
```
and change this line in the job definition to:
```
want_sles12sp2={want_sles12sp2}
```

**Comment by jdsn** [comment]
This is my proposal:
https://github.com/jdsn/SUSE-Cloud--automation/tree/jdsn/pr/1083

File: scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-crowbar_register-template.yaml (hunk 1 above)
Comment (dirkmueller) [inline scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-crowbar_register-template.yaml]: yeah, thats possible indeed.

**Comment by jdsn** [comment]
:+1:

**Pull request merged**